app.rb

```diff
@@ -0,0 +1,91 @@
+require_relative './book'
+require_relative './student'
+require_relative './teacher'
+require_relative './rental'
+
+class App
+  attr_reader :books, :people, :rentals
+
+  def initialize()
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def add_book
+    print 'Title:'
+    title = gets.chomp
+    print 'Author:'
+    author = gets.chomp
+    book = Book.new(title, author)
+    puts 'Book created successfully'
+    @books << book
+  end
+
+  def list_of_book
+    @books.each do |book|
+      puts "Title: #{book.title}, Author: #{book.author} "
+    end
+  end
+
+  def add_person
+    puts 'Add a person'
+    print 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
+    person_type = gets.chomp.to_i
+
+    add_person while person_type != 1 && person_type != 2
+    print 'Age: '
+    age = gets.chomp
+    print 'Name:'
+    name = gets.chomp
+
+    if person_type == 1
+      print 'Has parent permission? [Y/N]:'
+      response = gets.chomp.downcase
+      parent_permission = response == 'y'
+      @people << Student.new(age, name, parent_permission)
+    else
+      print 'Specialization: '
+      specialization = gets.chomp
+      @people << Teacher.new(age, specialization, name)
+    end
+    puts "Person created successfully \n"
+  end
```

Comment:
- [ ] Your project is looking great, you are almost done! However, I got an error when I tried to create a `person`:

<img width="985" alt="Screen Shot 2022-07-30 at 7 30 00 PM" src="https://user-images.githubusercontent.com/83717285/182004660-3316df2a-2faa-4a6a-bbf5-2cfa022149fd.png">

Let's fix that bug and you will be ready! 😎